MetaMask/metamask-extension
# Disable Bridge button on unsupported networks

## Explanation
* Disables the Bridge button when active network is unsupported and displays "Unavailable on this network" tooltip on hover
* Fixes [#18193](https://github.com/MetaMask/metamask-extension/issues/18193)




## Screenshots/Screencaps



### Before
<img width="359" alt="Screenshot 2023-03-21 at 12 09 04 PM" src="https://user-images.githubusercontent.com/100321200/226719452-b313687c-5445-4716-b371-b29b1b391f03.png">



### After
<img width="360" alt="Screenshot 2023-03-21 at 12 08 26 PM" src="https://user-images.githubusercontent.com/100321200/226719504-7f38309c-1108-48bb-945f-3faf1a83bb69.png">
<img width="356" alt="Screenshot 2023-03-21 at 12 08 31 PM" src="https://user-images.githubusercontent.com/100321200/226719517-34115421-bc51-497e-908b-af63789b5b18.png">



## Manual Testing Steps
1. Open the extension
2. Select the Ethereum network
3. Verify that Bridge button is enabled and opens the portfolio page when clicked
4. Open the extension
5. Select the Fantom network
6. Verify that the Bridge button is disabled (greyed out and unclickable)
7. Hover over the Bridge button and verify that "Unavailable on this network" is visible




## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
